index.html

```diff
@@ -798,13 +798,29 @@ <h3>
         button.</i>
       </p>
     </section>
-    <section id="permission-policy" data-cite="Feature-Policy">
+    <section>
+      <h2>
+        Extensions to the <code>WindowEventHandlers</code> Interface Mixin
+      </h2>
+      <p>
+        This specification extends the {{WindowEventHandlers}} interface mixin
+        from HTML to add [=event handler IDL attributes=] to facilitate the
+        event handler registration.
+      </p>
+      <pre class="idl">
+        partial interface mixin WindowEventHandlers {
```

Comment:
Should this be `partial interface Window` instead, because we don't really want to fire them on `document.body`? @marcoscaceres

Reply:
Yes, I think you are right.

Reply:
Checked some precedents and it seems anything using `partial interface Window` for adding event handlers are deprecated. (from https://w3c.github.io/deviceorientation/#idl-index, WebVR and some obsolete specs). Doing `WindowEventHandlers` also allows `<body ongamepadconnected="">` which is probably a good thing.

Reply:
BTW I submitted a patch, is any other thing blocking merging here?

Reply:
No blockers - just need ok from @nondebug or @JamesHollyer to confirm this matches Chrome.

Reply:
Also, could you kindly check what the merge conflict is?

Reply:
This looks fine to me. Chrome strangely has no IDL for the Gamepad event handlers, but this matches the IDL for other Window events.

Reply:
@nondebug, you are ok to add this to Chrome, right? Would you like me to file a bug for it?

Reply:
Filed https://crbug.com/1192878 to fix this in Chrome